torch/csrc/jit/codegen/cuda/ir_internal_nodes.h

```diff
@@ -277,6 +277,9 @@ struct TORCH_CUDA_API IterDomain : public Val {
     return start_;
   }
   Val* extent() const;
+  Val* rawExtent() const {
```

Comment:
nit: noexcept?

Reply:
As an optimization it's not helping much (most every C++ compiler would infer this from the definition, which is even inline here)

I'm a bit concerned about adding noexcept everywhere since it can actually add an overhead (compiler must generate code to check & enforce it unless it can prove it's not needed). 

I can add it if you insist, although it creates a slippery slope: in the absence of a detailed set of guidelines, the most it's used the more it encourages overuse, which in turn creates more problems than it solves (in particular, the unfortunate choice to have TORCH_CHECK report failures as exceptions makes it hard to predict where exceptions can originate from  a cursory review)

TL;DR: noexcept is a interface contract specification, not an optimization hint.